README.md

```diff
@@ -40,17 +40,17 @@ You should also be able to use the module directly in modern browsers supporting
 
 ```typescript
 // Load module
-import init from "@tableland/sqlparser";
+import { init } from "@tableland/sqlparser";
```

Comment:
Wait, doesn't the default import also work?

Reply:
It works if you're using some kind of tooling that enables synthetic default imports.  If you just try to run this code natively, e.g. using node.js, you will get an error saying `TypeError: init is not a function`

Reply:
I tried running this out of curiosity and I had to use `import { init }`. We can leave it as the default, but maybe add a comment on how to use it?

Reply:
Huh, that's probably some issue with our type defs. It should work. But I _much_ prefer explicit imports like this anyway.